temporian/implementation/numpy/operators/tests/assign/test_data/different_index.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""PandasAssignOperator - different index test.
+
+Tests that the assignee and assigned events cannot have different indexes.
+"""
+import pandas as pd
+
+from temporian.implementation.pandas.data.event import PandasEvent
+
+INPUT_1 = PandasEvent(
```

Comment:
Should it be Numpy event?

Reply:
Yeah, this script should be empty 😅 good catch!